rts/Sim/Path/QTPFS/Path.h

```diff
@@ -73,13 +82,15 @@ namespace QTPFS {
 		unsigned int GetNumPathUpdates() const { return numPathUpdates; }
 
 		void SetHash(std::uint64_t hash) { this->hash = hash; }
+		void SetVirtualHash(std::uint64_t virtualHash) { this->virtualHash = virtualHash; }
 		void SetRadius(float radius) { this->radius = radius; }
 		void SetSynced(bool synced) { this->synced = synced; }
 		void SetHasFullPath(bool fullPath) { this->haveFullPath = fullPath; }
 		void SetHasPartialPath(bool partialPath) { this->havePartialPath = partialPath; }
 
 		float GetRadius() const { return radius; }
 		std::uint64_t GetHash() const { return hash; }
+		std::uint64_t GetVirtualHash() const { return virtualHash; }
```

Comment:
consider
```suggestion
		std::uint64_t GetVirtualHash() const { return virtualHash; } // the difference from the real hash is XYZ
```

Reply:
> consider

I've added a desciption to the declaration further down.